### PR TITLE
Testeng 1567

### PR DESCRIPTION
**Pull request by matthew-wiemer**
This is needed for the unit testing of the interceptors. 

Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework/blob/master/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.1.1] - 2021-09-20
```
Comment (jonatanjakoniuk) [inline CHANGELOG.md:8]: it rather should be 6.2.0

**Comment by jonatanjakoniuk** [comment]
do we need 
` source/Relativity.Testing.Framework.Documentation/log.txt `
?

File: CHANGELOG.md (hunk 1 above)
Comment (matthew-wiemer) [reply CHANGELOG.md:8]: I had argument against this when I went from 6.0.0 to 6.0.1.

I'm not exposing any functionality the client didn't have access to before. I'm just making it available on an interface. Seemed like more of a design bug to me than "adding functionality". Does that change your mind?

**Comment by matthew-wiemer** [comment]
> do we need
> `source/Relativity.Testing.Framework.Documentation/log.txt`
> ?

No good catch, that slipped in.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.2.0] - 2021-09-20
+
+### Added
+
+- IApplicationInsightsTelemtryClient now has a TrackEvent method that accepts a <string, double> dictionary of metrics.
```
Comment (briantoms) [inline CHANGELOG.md:12]: `Telemtry` -> `Telemetry`

- Fix typo

File: source/Relativity.Testing.Framework/IApplicationInsightsTelemetryClient.cs [hunk 3]
```diff
@@ -29,6 +29,14 @@ public interface IApplicationInsightsTelemetryClient
 		/// <param name="properties">A related dictionary of properties to associate with the event.</param>
 		void TrackEvent(string metricName, Dictionary<string, string> properties);
 
+		/// <summary>
+		/// Sends an arbitrary event with related metrics to the configured telemetry client.
+		/// </summary>
+		/// <param name="metricName">The name of the event being sent.</param>
+		/// <param name="properties">A related dictionary of properties to associate with the event.</param>
+		/// <param name="metrics">A relevant set of metrics associated with the event.</param>
+		void TrackEvent(string metricName, Dictionary<string, string> properties, Dictionary<string, double> metrics);
```
Comment (briantoms) [inline source/Relativity.Testing.Framework/IApplicationInsightsTelemetryClient.cs:38]: How will this be used? I don't immediately see how this is needed for the story in question.

Comment (matthew-wiemer) [reply source/Relativity.Testing.Framework/IApplicationInsightsTelemetryClient.cs:38]: I didn't either, but the `ApplicationInsightsEventsInterceptor` actually takes a `metrics` parameter that is a dictionary of a single parameter containing the invocation time. 

https://github.com/relativitydev/relativity.testing.framework.api/blob/master/source/Relativity.Testing.Framework.Api/Interceptors/ApplicationInsightsEventInterceptor.cs#L48

**Review by briantoms — APPROVED** [review]

**Pull request merged**